packages/@aws-cdk/aws-lambda/lib/layers.ts

```diff
@@ -28,6 +28,14 @@ export interface LayerVersionOptions {
    * @default - A name will be generated.
    */
   readonly layerVersionName?: string;
+
+  /**
+   * Whether to retain the previous version of the layer when a new version is added
+   * or when the stack is deleted.
+   *
+   * @default - The previous version is deleted.
```

Comment:
I believe it should be this -

```suggestion
   * @default RemovalPolicy.DESTROY
```